build/buildconstants/drand.go

```diff
@@ -10,3 +16,91 @@ const (
 	DrandIncentinet
 	DrandQuicknet
 )
+
+type DrandConfig struct {
```

Comment:
@ribasushi Please can you leave a comment explaining these large DRAND related diffs ?